grouper/fe/templates/permission-request-update.html

```diff
@@ -30,8 +30,12 @@
                 </tr>
                 <tr>
                     <td colspan="5">
-                        <span style="padding-left: 10px;">
-                            <strong>Reason:</strong> <em>{{comment.comment}}</em>
+                        <dl class="dl-horizontal" style="margin-bottom: 0px;">
```

Comment:
i copied/pasted from the join request flow. so yeah, probably a TODO for cleanup.